src/hapi/pipelines/utilities/operational_presence.py

```diff
@@ -148,20 +152,51 @@ def populate(self, metadata: Metadata):
         self._session.commit()
 
     def _get_org_type_code(self, org_type: str) -> str:
-        # TODO: implement fuzzy matching of org types
-        org_type_data = self._org_type.data
-        org_type_code = org_type_data.get(org_type, "")
+        org_type = normalize_string(org_type)
+        org_type_code = self._org_type.data.get(org_type)
+        if not org_type_code:
+            org_type_code = self._org_type_map.get(org_type.lower())
+        if not org_type_code:
+            # TODO: implement fuzzy matching of org types
+            org_type_code = ""
         return org_type_code
 
-    def _get_sector_info(self, sector_info: str, info_type: str) -> (str, str):
-        # TODO: implement fuzzy matching of sector names/codes
+    def _get_sector_info(
+        self, sector_name_start: str, sector_code_start: str
+    ) -> (str, str):
         sector_data = self._sector.data
         sector_names = {name: sector_data[name] for name in sector_data}
         sector_codes = {sector_data[name]: name for name in sector_data}
 
-        if info_type == "name":
-            sector_code = sector_names.get(sector_info, "")
-            return sector_code
-        if info_type == "code":
-            sector_name = sector_codes.get(sector_info, "")
-            return sector_name
+        if not sector_name_start and not sector_code_start:
```

Comment:
I realized that I don't totally understand the different between `sector_code` and `sector_name`. I know that in the config file you put NGA as having `code`, and AFG and MLI as having `name`. However looking through the specific values in the datasets I don't see too much difference - e.g. NGA doesn't use any abbreviations, while MLI does use a few. 

Would it be possible to combine them, or are there 3W files that contain both?

Reply:
Good idea, I'll combine them.  There are files that contain both, but there's a lot of overlap between the two so it won't make much of a difference in the matching to official sectors.